specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1616,7 +1623,7 @@
           "description": "The target environment assertion."
         }
       },
-      "description": "The export key parameters.",
+      "description": "The release key parameters.",
```

Comment:
The key release parameters rather than release key parameters

Reply:
I think its equivalent ;-)